GameData/ROEngines/RealPlume/XLR99_CH4_RealPlume.cfg

```diff
@@ -0,0 +1,37 @@
+//	==================================================
+//	XLR99 Series engine plume setup.
+//	==================================================
+
+@PART[ROE-XLR99]:FOR[RealPlume]:NEEDS[SmokeScreen]
```

Comment:
Do not declare `:FOR[AnotherMod]`.  Change this FOR+NEEDS to just `:BEFORE[RealPlume]`